cmd/nginx/flags.go

```diff
@@ -213,47 +208,35 @@ Feature backed by OpenResty Lua libraries. Requires that OCSP stapling is not en
 		glog.Warningf("SSL certificate chain completion is disabled (--enable-ssl-chain-completion=false)")
 	}
 
-	if (*enableSSLChainCompletion || !*dynamicConfigurationEnabled) && *dynamicCertificatesEnabled {
+	if *enableSSLChainCompletion && *dynamicCertificatesEnabled {
 		return false, nil, fmt.Errorf(`SSL certificate chain completion cannot be enabled and dynamic configuration cannot be disabled when 
 dynamic certificates functionality is enabled. Please check the flags --enable-ssl-chain-completion and --enable-dynamic-configuration`)
 	}
 
-	// LuaJIT is not available on arch s390x and ppc64le
-	disableLua := false
-	if runtime.GOARCH == "s390x" || runtime.GOARCH == "ppc64le" {
```

Comment:
So this mean we are also officially dropping support for `s390x` and `ppc64le` as well right? If so can you also cleanup make and build files?

Reply:
No, there is luajit support for ppc64le. I think I will disable s390x and ask for users with access to that hardware if they can provide some kind of access.